R/anova.rlm.R

```diff
@@ -0,0 +1,49 @@
+## Purpose:   robust F-test: Wald test for several coefficients of
+##            an rlm object
+## -------------------------------------------------------------------------
+## Arguments:
+##   object   result of rlm(...)
+##   var      variables. Either their names or their indices
+##		  Default: all *but* the intercept
+## -------------------------------------------------------------------------
+## Author: Werner Stahel, Date: 14 Jul 2000;  MM, 2000-07-14
+##- From: Werner Stahel <stahel@stat.math.ethz.ch>
+##- To: holzer@stat.math.ethz.ch, maechler@stat.math.ethz.ch
+##- Subject: robust F-test
+##- Date: Fri, 14 Jul 2000 17:01:55 +0200 (CEST)
+
+## Taken from sfsmisc package 
```

Comment:
Don't include it if it's taken from another package, just use the other package instead.

In this case, the function has since been renamed to `f.robftest`, which is a better name for it because this is not `anova` (which by itself handles `rlm` objects fine, just doesn't do F-tests).